pipeline/src/scripts/testGraphQuery.ts

```diff
@@ -72,21 +90,77 @@ async function main() {
           id: id || 'Wsttgx8AAJeSNmJ4',
         };
       case 'exhibition':
-        return { graphQuery: '', id: id || 'Yzv9ChEAABfUrkVp' };
+        return {
+          graphQuery: isAddressable
+            ? `{
+          ${addressablesExhibitionsQuery}
+          }`
+            : '',
+          id: id || 'Yzv9ChEAABfUrkVp',
+        };
       case 'book':
-        return { graphQuery: '', id: id || 'ZijgihEAACMAtL-k' };
+        return {
+          graphQuery: isAddressable
+            ? `{
+            ${addressablesBooksQuery}
+          }`
+            : '',
+          id: id || 'ZijgihEAACMAtL-k',
+        };
       case 'page':
-        return { graphQuery: '', id: id || 'YdXSvhAAAIAW7YXQ' };
+        return {
+          graphQuery: isAddressable
+            ? `{
+          ${addressablesPagesQuery}
+          }`
+            : '',
+          id: id || 'YdXSvhAAAIAW7YXQ',
+        };
       case 'visual-story':
-        return { graphQuery: '', id: id || 'Zs8EuRAAAB4APxrA' };
+        return {
+          graphQuery: isAddressable
+            ? `{
+          ${addressablesVisualStoriesQuery}
+          }`
+            : '',
+          id: id || 'Zs8EuRAAAB4APxrA',
+        };
       case 'exhibition-text':
-        return { graphQuery: '', id: id || 'Zs8mohAAAB4AP4sc' };
+        return {
+          graphQuery: isAddressable
+            ? `{
+            ${addressablesExhibitionTextsQuery}
+          }`
+            : '',
+          id: id || 'Zs8mohAAAB4AP4sc',
```

Comment:
This is not for this ticket and is a very general ponder; should we move it the other way around since most of these are Addressables? The flag being `isSpecific` or something instead.

Reply:
If you think it's a good idea I'm happy to do it as part of https://github.com/wellcomecollection/content-api/pull/161